collection/alex.json

```diff
@@ -17,7 +17,9 @@
 		"ces": "alexej",
 		"spa": "alejo",
 		"lat": "alexius",
-		"heb": "אלכס"
+		"heb": "אלכס",
+		"ro" : "Alex"
```

Comment:
Hey @hackmajoris,
similar to your other PR #509 please make sure you use the ISO-639-3 language codes and the translation is all lowercased. That should fix the failing tests.